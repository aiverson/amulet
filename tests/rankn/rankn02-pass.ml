let foo : ( (forall 'a. 'a -> 'a) -> bool ) * int =
  ( (fun _ -> false) : (forall 'b. lazy 'b -> lazy 'b) -> bool
  , 0
  )
