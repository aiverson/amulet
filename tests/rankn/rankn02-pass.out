foo : ({'a : type}. 'a -> 'a) -> bool * int
